public/components/overview/mitre/components/techniques/techniques.tsx

```diff
@@ -281,6 +303,13 @@ export const Techniques = withWindowSize(class Techniques extends Component {
     this.props.onSelectedTabChanged('dashboard');
   }
 
+  openIntelligence(e,redirectTo,itemId){
+    sessionStorage.setItem("tabRedirect", redirectTo);
+    sessionStorage.setItem("idToRedirect", itemId);
+    this.props.onSelectedTabChanged('intelligence');
+    window.location.href = window.location+`&tabRedirect=${redirectTo}&idToRedirect=${itemId}`
+  }
```

Comment:
why it is storing and sending in the URL the same data?

Reply:
the storing was not used, now I delete it